cookiecutter-django-app/{{cookiecutter.repo_name}}/.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish package to PyPi
```

Comment:
Were you not able to get this from the python-template? This seems to be an exact copy from python-template

Reply:
I thought python-template was only used as a base for python-library, you're right we don't this here.